.gitignore

```diff
@@ -8,3 +8,7 @@ spec/dummy/log/*.log
 spec/dummy/public/assets
 spec/dummy/tmp/
 spec/dummy/.sass-cache
+
+# bower-rails
```

Comment:
It gets automatically generated anyway. So committing `.bowerrc` but not `bower.json` seems inconsistent.